plugins/LilyPad/LilyPad.cpp

```diff
@@ -384,8 +384,15 @@ void AddForce(ButtonSum *sum, u8 cmd, int delta = 255)
 
 void ProcessButtonBinding(Binding *b, ButtonSum *sum, int value)
 {
-    if (value < b->deadZone || !value)
+    if (value < b->deadZone || !value) {
```

Comment:
Well, it's an ``int``, so ``value == 0`` might be more suitable but just my opinion. :)